src/main/java/net/socialgamer/cah/handlers/FirstLoadHandler.java

```diff
@@ -68,22 +69,26 @@
   private final Set<String> banList;
   private final Session hibernateSession;
   private final Provider<Boolean> includeInactiveCardsetsProvider;
+  private final boolean globalChatEnabled;
```

Comment:
Just noticed this now, but this needs to be a Provider<Boolean> since that configuration value can change at runtime (and for my only server that does have it enabled, it will change from false to true at runtime). I'll go ahead and fix this myself.

Reply:
No, on second look, it doesn't, since a new FirstLoadHandler is created every time. Neither of those need to be providers, so I'm ripping that back out since I'm in this file making other changes (which is what made me stop to think about it again). It is obvious that a new one is created every time since we inject a Hibernate session (and not a provider) and we close that session at the end of the handler.